docs/raw/9-blockchain-interactions.md

```diff
@@ -0,0 +1,186 @@
+---
+permalink: /spec/9
+parent: Raw specs
+title: 9/BLOCKCHAIN-USAGE
+---
+
+# 3/BLOCKCHAIN-USAGE
+
+> Version: 0.1
+>
+> Status: Raw
+>
+> Authors: Andrea Maria Piana <andreap@status.im>
+
+# Status interactions with the blockchain
+
+In this document is document all the interactions that the Status client has
+with the blockchain.
+
```

Comment:
Be explicit about pre-requisites

e.g. use a node, or use infura

For each mode what requirements do we put on it? E.g.

- trust source? integrity of transactions
- key handling (link to other spec for how those properties are preserved)
- availability assumptions? e.g. if infura is blocked you can't transact
- rate limiting considerations - e.g. if we need to query historical tx that might be suboptimal for <reasons> (indexing, etc)